### PR TITLE
Add 2000190 as a CH patient

### PR DESCRIPTION
**Pull request by KevinCranmer**

### CODE REVIEW

**Review by bcastillomartel-va — APPROVED** [review]

File: datamart/README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Non-Synthea ICNs:
   - `25000126`
   - `32000225`
   - `5000335`
+  - `2000190`
```
Comment (jblefkove-va) [inline datamart/README.md:14]: Above, is it now 78/83 ?

File: datamart/README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ Non-Synthea ICNs:
...
 
 - Clipboard
```
Comment (jblefkove-va) [inline datamart/README.md:16]: Would this be `Clipboard & Dr. First` now?

Comment (KevinCranmer) [reply datamart/README.md:16]: It will be once we get Bryan's synthea transformer PR completed

Comment (KevinCranmer) [reply datamart/README.md:16]: Well I guess, it actually is still used as Dr. First

**Review by bcastillomartel-va — APPROVED** [review]

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**